python/ray/autoscaler/updater.py

```diff
@@ -57,8 +58,9 @@ def __init__(self,
                  exit_on_update_fail=False,
                  use_internal_ip=False):
 
-        ssh_control_path = "/tmp/{}_ray_ssh_sockets/{}".format(
```

Comment:
Is there a reason for removing the "_sockets" suffix? Doesn't seem like a big deal but not necessary either.

Reply:
Yeah, there is still the user name in the path (which we should keep so there is no clash between users) and I made this change so it is as robust as possible against long user names. If we wanted to be paranoid about this, we could hash the user name too if it is too long, but this is probably fine.

Reply:
I'm now also hashing the user name, to make it super robust ;)